lib/apiservers/engine/backends/container.go

```diff
@@ -238,7 +238,7 @@ func (c *Container) ContainerExecCreate(name string, config *types.ExecConfig) (
 		OpenStdin: config.AttachStdin,
 		Tty:       config.Tty,
 	}
-	log.Debugf("JoinConfig: %#v", joinconfig)
+	log.Infof("JoinConfig: %#v", joinconfig)
 
 	// obtain a portlayer client
 	client := c.containerProxy.Client()
```

Comment:
For consistency I'd prefer this go into container_proxy. If we need to modify `CreateContainerTask` so that it can be used for both the base container tasks and exec'd tasks then we should.